kafka-client-examples/simple-example/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id("responsive.java-application-conventions")
+    id("responsive.docker")
+}
+
+application {
+    mainClass.set("dev.responsive.examples.simpleapp.Main")
+}
+
+dependencies {
+    // todo: how to set the version here?
+    implementation(project(":kafka-client"))
```

Comment:
I think the only way to do this would be to depend on published artifacts (e.g. `"implementation("dev.responsive...:kafka-client:<version>")`)

Reply:
This is not an issue anymore. Just forgot to cleanup the comment. Getting the client version built in the same build is what we want.